src/CMakeLists.txt

```diff
@@ -128,12 +128,20 @@ IF(GIT_FOUND AND IS_DIRECTORY "${CMAKE_SOURCE_DIR}/.git")
 	                OUTPUT_VARIABLE VERSION_STRING
 	                WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
 	)
-	string(STRIP "${VERSION_STRING}" VERSION_STRING) # Newline
+	string(STRIP "0.1~git${VERSION_STRING}" VERSION_STRING) # Newline
+ELSEIF(IS_DIRECTORY "${CMAKE_SOURCE_DIR}/.bzr")
+	execute_process(COMMAND bzr revno
+	                OUTPUT_VARIABLE VERSION_STRING
+	                WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
+	)
+	string(STRIP "0.1~brz${VERSION_STRING}" VERSION_STRING) # Newline
```

Comment:
Not sure why you added this.

I used `bzr` for a while, but that is ages ago. I am fine if we just drop this (but fine if you want to keep it as well).

I also used `hg` and I think that VCS used to be a lot bigger, yet you decided not to add it (again, I am fine either way, just curious why you made these choices).

Reply:
Removed it again. For launchpad mirroring bzr is needed, but since the PPA wants to specify a custom version string anyway this was unnecessary.